### PR TITLE
I18n/fr: fix some errors

### PR DESCRIPTION
**Pull request by Letiste**

### CODE REVIEW

**Comment by bastimeyer** [comment]
Thanks a lot for the fixes, looks good!

Since I can't do it myself, would you please be so kind and take a look at the remaining missing French translations and add them real quick? New stuff has been added to the app over the past months and only the `fr` locale is currently still missing new translations because nobody has added them yet.

The missing translation keys can be seen by running `yarn run grunt webpack:i18n` or in the output of the i18n step of Github's test runners:
https://github.com/streamlink/streamlink-twitch-gui/runs/1165021180#step:9:13

These translation keys are compared against the `en` locale and they need to be copied from the respective yml files from the `en` locale to the yml files of the `fr` locale, at the same relative positions where they are in the `en` files.

I would really appreciate if you could do this, so that there are no more missing translations. Thank you!

**Comment by Letiste** [comment]
Sure ! I saw some translations missing but didn't know how to add them. 
Do I need to create a new pull request or can I just add the changes to this pull request?

**Comment by bastimeyer** [comment]
If you prefer opening a new PR for the missing translations, then you can do it of course. I would also be fine with adding them here. I will squash the commits on merge though, so if you want to keep the fixes and addition of missing translations separate, open a new PR.

**Review by bastimeyer — CHANGES_REQUESTED** [review]
Thanks a lot! The changes are looking good and all missing translations have been added. There's one tiny issue though which needs to be fixed before I can merge this. Could you please take a look?

File: src/app/locales/fr/settings.yml [hunk 1]
```diff
@@ -3,10 +3,11 @@ menu:
     main: Principal
     gui: Interface
     streaming: Streaming
-    player: Joueur
+    player: Lecteur
     streams: Streams
     chat: Chat
     languages: Langages
+    hotkeys: Raccourcis claviers
```
[the remark this answers is not in the document]
Comment (Letiste) [reply src/app/locales/fr/settings.yml:10]: "Raccourcis"  should be fine and doesn't break the submenu (at least for me).

**Comment by bastimeyer** [comment]
Yep, that fixes the settings submenu.

I'll leave the PR up for another day or so, just in case somebody wants to review or if you find something else to fix. Then I'm going to merge.

**Comment by bastimeyer** [comment]
Thanks!

**Pull request merged**